docs/md/views/selection-table.md

```
@@ -0,0 +1,25 @@
+
+In essence this is the bridge between selecting cross-links/matches in a view and accessing the underlying data in the Spectrum View.
+
+Selection of cross-links and/or matches in a view populates this paged table, filling the bottom part of the display above the filter bar - the height of the displayed area is adjustable with a draggable split pane widget.
+
+The table also reacts to filtering, as the matches shown in the table have to be both selected and pass the current filter state.
+
+### Representation ###
+
+In the table, matches are shown as rows which are divided into groups according to the cross-link they belong to, with cross-links in turn sorted by the score of their highest-scoring match. Ambiguous matches are repeated in each cross-link they could belong to. The title bar of the table states the number of currently selected cross-links and matches with a paging widget for tabulating through large collections of selected matches.
+
+### Interaction
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: atx_closed; Actual: atx

(MD003, heading-style)